Khan/perseus
# Create helpers to replace snowman regex uses on the front-end

## Summary:
There are several locations in webapp where regexes are used to extract widget placeholder helpers or related information from Perseus JSON. These helpers should allow us to remove those regexes, providing us with more flexible when changing structures in Perseus. It also allows for easier testing when related changes are made.

Issue: LEMS-1800

## Test plan:
- Confirm all checks pass
- Use npm snapshot to confirm these helpers can correctly replace regex uses in webapp
